README.rst

```diff
@@ -79,12 +79,17 @@ You need a VIM version that was compiled with Python 2.7 or later
 
 Manual installation
 -------------------
+The plugin uses `git` submodules to fetch latest version of Jedi.
 
-You might want to use `pathogen <https://github.com/tpope/vim-pathogen>`_ or
-`Vundle <https://github.com/gmarik/vundle>`_ to install jedi-vim.
+To install using standard Vim 8 plugin system:
+
+.. code-block:: sh
 
-The first thing you need after that is an up-to-date version of Jedi. Install
-``git submodule update --init --recursive`` in your jedi-vim repository.
+    mkdir -p ~/.vim/pack/git-plugins/start
+    git clone --recursive https://github.com/davidhalter/jedi-vim.git ~/.vim/pack/git-plugins/start/jedi-vim
```

Comment:
btw: do you know if there is a preferred/recommended way to update Vim packages?

Reply:
no :D

Reply:
@abitrolly do you know?  How do you keep them updated?

(as for me: I am using vim-plug basically, but given the amount of local changes I still update them manually in general, i.e. by updating/merging into the Git checkout)

Reply:
I update with plain `git`.
```
git -C ~/.vim/pack/git-plugins/start/jedi-vim up
```